src/core/jupyter/jupyter.ts

```diff
@@ -1719,8 +1719,22 @@ function mdOutputStream(output: JupyterOutputStream) {
   return mdCodeOutput(output.text.map(colors.stripColor));
 }
 
-function mdOutputError(output: JupyterOutputError) {
-  return mdCodeOutput([output.ename + ": " + output.evalue]);
+async function mdOutputError(
+  output: JupyterOutputError,
+  options: JupyterToMarkdownOptions,
+) {
+  if (!options.toHtml || !hasAnsiEscapeCodes(output.evalue)) {
+    return mdCodeOutput([output.ename + ": " + output.evalue]);
+  }
+  const html = await convertToHtmlSpans(output.evalue);
+  console.log(output.evalue);
```

Comment:
Is this `console.log(output.evalue)` required?

Reply:
Nope, that's me being stupid. Thanks for the catch.

Reply:
Fixed on `main`.

Reply:
Awesome thanks!